_regenerate_json.sh

```diff
@@ -43,14 +69,7 @@ do
     fi
 
     # We strip error lines because of CFE-2370 and CFE-2696
-    ${WRKDIR}/core/cf-promises/cf-promises --eval-functions --policy-output-format=json ${policy} | sed '/   error\:.*/d' >  ${out}
-    # Do we need --eval-functions?
-    echo "Writing '${out}'"
-    printf '%0.1s' "-"{1..60}
-    echo
-    cat "${out}"
-    printf '%0.1s' "-"{1..60}
-    echo
+    "$cf_promises" --eval-functions --policy-output-format=json ${policy} | sed '/   error\:.*/d' >  ${out}
```

Comment:
I am unsure about removing this debugging stuff here, I think it may have been added to surface syntax errors and so that we would know which file the error was in.